src/api/app/models/user.rb

```diff
@@ -42,6 +42,7 @@ class User < ApplicationRecord
 
   belongs_to :owner, class_name: 'User'
   has_many :subaccounts, class_name: 'User', foreign_key: 'owner_id'
+  has_many :bs_requests
```

Comment:
I guess bs_requests is fine, but I somewhat would like to call it created_bs_requests since that also defines the kind of relation.
But I would ask for some other opinions here.